forte/ontology_specs/ag_news.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "ag_news",
+    "imports": [
+      "base_ontology.json"
+    ],
+    "definitions": [
+      {
+        "entry_name": "ft.onto.ag_news.Article",
```

Comment:
Interesting, so you choose to extend the base ontology. This is a good idea! One thing I would suggest is that we can actually add a `document_class` attribute the the `base_ontology`, since it is common to have a category for a document.  To make things general, you may consider to use a list of strings for that variable. In the AG news corpus, we happen to use just one of them.